osu.Game/Screens/Edit/Components/Timelines/Summary/Parts/GroupVisualisation.cs

```diff
@@ -60,5 +57,13 @@ protected override void LoadComplete()
                 }
             }, true);
         }
+
+        /// <summary>
+        /// For display purposes, check whether the proposed group is made redundant by this visualisation group.
+        /// </summary>
+        /// <param name="other"></param>
+        /// <returns></returns>
```

Comment:
nit: similar xmldoc should go on the `IControlPointVisualisationRedundant` interface member

Reply:
Have added and fixed the empty sections here.